results.go

```diff
@@ -1281,6 +1286,72 @@ func (rh *ResultsHandler) uploadHandler(w http.ResponseWriter, r *http.Request)
 	http.Redirect(w, r, r.Referer(), http.StatusFound)
 }
 
+type combineResultsTemplateVars struct {
+	BaseTemplateVars
+
+	Results []SessionResults
+}
+
+const combinedSuffix string = "-combined"
+
+func (rh *ResultsHandler) combineResults(w http.ResponseWriter, r *http.Request) {
+	if r.Method == http.MethodPost {
+		if err := r.ParseForm(); err != nil {
+			logrus.WithError(err).Error("Combine Results: Couldn't parse form")
+			http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
+			return
+		}
+
+		CombineResultsSessionFiles := r.Form["CombineResults"]
```

Comment:
can this be combineResultsSessionFiles? it isn't (and can't be) a public variable because it's inside a method, so no need for the uppercase C